README.md

```diff
@@ -105,38 +103,63 @@ React.render(<App person={person} />, document.body)
 person.name = "Mike" // will cause the Observer region to re-render
 ```
 
-Example2:
+In case you are a fan of render props, you can use that instead of children. Be advised, that you cannot use both approaches at once, children have a precedence.
+
+Here goes example showing render prop only
```

Comment:
Well, I did not mean this line to be included there as well, that's why I used italic ;)